packages/palette/src/elements/Message/Message.tsx

```diff
@@ -29,14 +29,15 @@ const StyledFlex = styledWrapper(Flex)`
  */
 export const Message: SFC<MessageProps> = ({
   children,
-  textSize = "3t",
+  textSize,
   ...others
 }) => {
   return (
     <StyledFlex p={2} {...others}>
-      <Sans size={textSize} color={color("black60")} weight="regular">
+      {/* TODO: Can the size be left here? */}
+      <Text size={textSize} color={color("black60")} variant="text">
```

Comment:
`size` is a valid prop from layout:

![](http://static.damonzucconi.com/_capture/PejAX0DVVf3j.png)

It's probably worth just denoting this a breaking change and removing `textSize` or accept `variant` or just ignore it completely?

Reply:
Ahh, crap, I see. Yeah, will need to figure out how to properly update this and find where it's used.

Reply:
Resolved this but it's a breaking change.